src/Linear/Annotations.tsx

```diff
@@ -182,12 +182,27 @@ const SingleNamedElement = (props: {
     }
   }
 
-  // determine whether the element name fits within the width of the element
-  const nameLength = name.length * 6.75; // aspect ratio of roboto mono is ~0.66
-  const nameFits = nameLength < width - 15;
+  // 0.591 is our best approximation of Roboto Mono's aspect ratio (width / height).
+  const annotationCharacterWidth = 0.591 * 12;
+  const availableCharacters = Math.floor((width - 40) / annotationCharacterWidth);
```

Comment:
where did the `-40` come from? Is that the equivalent of 20px padding on both sides?